worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -372,6 +411,55 @@ public Void call() throws Exception {
         }
     }
 
+    private static class SchematicShareTask implements Callable<URL> {
+        private final Actor actor;
+        private final ClipboardFormat format;
+        private final ClipboardHolder holder;
+        private final String name;
+
+        SchematicShareTask(Actor actor, ClipboardFormat format, ClipboardHolder holder, String name) {
+            this.actor = actor;
+            this.format = format;
+            this.holder = holder;
+            this.name = name;
+        }
+
+        @Override
+        public URL call() throws Exception {
+            Clipboard clipboard = holder.getClipboard();
```

Comment:
Can we merge the saving code here and with `save` into a single method? DRY applies here, especially with how much custom logic is part of saving.